project_silentdata.md

```diff
@@ -0,0 +1,188 @@
+
+LinkedIn: <https://www.linkedin.com/in/rseromenho/>
+
+GitHub: <https://github.com/seromenho>
+
+**Thomas Brooks** is one of the core developers on the Silent Data platform at Applied Blockchain. He joined the company after completing a doctorate in high energy particle physics, developing software and machine learning algorithms for 3D particle interaction reconstruction. Thomas has a broad range of experience in software development, from creating open source visualisation and analysis tools used by world leading biology labs to writing Ethereum smart contracts for DeFi applications.
+
+LinkedIn: <https://www.linkedin.com/in/tom-brooks-a940a9a7/>
+
+GitHub: <https://github.com/tgrbrooks>
+
+**Shay Har-Zion** is a product manager with over 17 years of experience in leading product and project software developments, with majority of them in the financial services and Fintech industries. Shay puts in front the focus on teamwork, communication and collaboration, in order to produce highest value and quality towards innovative solutions for clients.
+
+### Team Code Repos
+
+- <https://github.com/appliedblockchain>
+- <https://github.com/appliedblockchain/silentdata-defi-core> (The SGX components)
+- <https://github.com/appliedblockchain/silentdata-defi-app> (The web app components)
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- <https://github.com/seromenho>
+- <https://github.com/tgrbrooks>
+
+### Team LinkedIn Profiles (if available)
+
+- <https://www.linkedin.com/in/adibenari/>
+- <https://www.linkedin.com/in/shay-har-zion-3a31933/>
+- <https://www.linkedin.com/in/andylnd/>
+- <https://www.linkedin.com/in/tgrbrooks/>
+
+## Development Status :open_book:
+
+If you've already started implementing your project or it is part of a larger repository, please **provide a link and a description of the code here**. In any case, please provide some documentation on the research and other work you have conducted before applying.
+
+Silent Data Architecture: [Silent Data Architecture](https://docs.google.com/document/d/1RXk8gy_YBtnWYgYXNh2lizZ94kXyBI7tLAmhxnvsYS0/edit?usp=sharing)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+### Milestone 1 — Substrate Adapter
+
+- Estimated duration: 2 months
+- FTE: 2.5
+- Costs: 30,000 USD
+
+
+| Number  | Deliverables                   | Specification                                                                   | 
+| ------- |:------------------------------:| -------------------------------------------------------------------------------:|
+| 0a.     | License                        | MIT.                                                                           
+| 0b.     | Documentation                  | We will provide documentation on Silent Data. 

```

Comment:
This specification is a bit too vague. Please describe what this looks like (e.g. inline documentation, setup guide, etc.)

Reply:
We already have a whitepaper describing the principles behind the Silent Data system, but we will also provide Polkadot specific setup instructions as well as supporting documentation (in the form of READMEs) for the example smart contract code.